app/web/src/store/func/funcs.store.ts

```diff
@@ -940,6 +947,39 @@ export const useFuncStore = () => {
               }
             },
           },
+          {
+            eventType: "ManagementOperationsComplete",
+            callback: (
+              { requestUlid, status, message, createdComponentIds, funcName },
+              _metadata,
+            ) => {
+              if (
+                requestUlid &&
+                requestUlid === this.pendingManagementRunRequestUlid &&
+                createdComponentIds?.length
+              ) {
+                const componentsStore = useComponentsStore();
+                componentsStore.eventBus.emit(
```

Comment:
i dont think we need all this event bus nonsense... the `setSelectedComponentIds` fn is now in the view store...